packages/server/package.json

```diff
@@ -123,8 +114,8 @@
     "graphql-shield": "^7.5.0",
     "html-minifier-terser": "^5.0.2",
     "immutable": "3.8.2",
-    "intersection-observer": "^0.7.0",
     "ioredis": "^4.11.2",
+    "jsdom": "^16.6.0",
```

Comment:
This was the only dependency I found that needed to be moved from `devDependencies` to `dependencies`. It is used at runtime to sanitize SVGs: https://github.com/ParabolInc/parabol/blob/master/packages/server/fileStorage/validateAvatarUpload.ts